python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -579,6 +579,37 @@ async def dome_el_in_position(self) -> str:
         await self._dome_el_in_position.wait()
         return "Dome elevation in position."
 
+    async def wait_for_dome_ready(self) -> None:
+        """Wait for the dome to be in a state where it can be moved."""
+        dome_state = await self.rem.mtdome.evt_summaryState.aget(
+            timeout=self.fast_timeout
+        )
+        if dome_state.summaryState != salobj.State.ENABLED:
+            raise RuntimeError("Dome is not enabled. Cannot move.")
+
+        self.log.info("Dome is ready for movement.")
+
+    async def park_dome(self) -> None:
+        """Park the dome by moving it to the park azimuth."""
+        self.log.info(f"Parking dome at azimuth {self.dome_park_az}.")
+
+        # Ensure the dome is ready for a move
+        await self.wait_for_dome_ready()
+
+        # Move the dome to the park position
+        await self.rem.mtdome.cmd_park.set_start(timeout=self.long_timeout)
```

Comment:
Use `start` instead of `set_start` if you are not setting any attribute besides timeout.

Reply:
bba97f5aae2eb4f18a5daf91961d9b935213a306

Reply:
f358ad671ca65d197dd14207905df0efe2b4930c